package.json

```diff
@@ -83,6 +84,8 @@
     "mocha": "3.2.0",
     "mocha-webpack": "0.7.0",
     "node-sass": "4.5.2",
+    "npm": "5.5.1",
+    "or": "0.2.0",
```

Comment:
What is this for?

Reply:
it seems to have been updated when I run npm install